### PR TITLE
Reference tree manager in theme selector docs

### PR DESCRIPTION
**Pull request by tsauerwein**
Closes https://github.com/camptocamp/ngeo/issues/1914

### CODE REVIEW

File: contribs/gmf/src/services/treemanager.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ goog.require('ngeo.StateManager');
  * 'add', children will be just pushed in this array. The default state can be
  * changed by setting the value `gmfTreeManagerModeFlush`, e.g.:
  *
- *    var module = angular.module('app');
- *    module.value('gmfTreeManagerModeFlush', false);
+ *     var module = angular.module('app');
+ *     module.value('gmfTreeManagerModeFlush', false);
```
Comment (sbrunner) [inline contribs/gmf/src/services/treemanager.js]: Between the two adds (desktop / desktop_alt) we also have some other changes:
```
            <a href class="btn btn-default btn-block btn-primary"
              data-toggle="dropdown">
              <span class="fa fa-grid"></span>
              <span translate>Theme:</span>
              <b ng-if="!mainCtrl.gmfThemeManager.themeName" translate>Loading...</b>
              <b ng-if="mainCtrl.gmfThemeManager.themeName">{{mainCtrl.gmfThemeManager.themeName|translate}}</b>
              <span class="caret"></span>
            </a>
```
=>
```
            <a href class="btn btn-default btn-block btn-primary"
              data-toggle="dropdown">
              <span class="fa fa-grid"></span>
              <span translate>Themes</span>
              <span class="caret"></span>
            </a>
```

Comment (sbrunner) [inline contribs/gmf/src/services/treemanager.js]: Minimal version:
```
              <span translate>Theme:</span>
              <b ng-if="!mainCtrl.gmfThemeManager.themeName" translate>Loading...</b>
              <b ng-if="mainCtrl.gmfThemeManager.themeName">{{mainCtrl.gmfThemeManager.themeName|translate}}</b>
```
=>
```
              <span translate>Themes</span>
```

**Comment by tsauerwein** [comment]
Updated. @sbrunner, please check if it is ok for you like that. If not, feel free to adapt my branch.

**Comment by sbrunner** [comment]
It miss `<span translate>Theme`**:**`</span>` => `<span translate>Theme`**s**`</span>` :-)

**Comment by tsauerwein** [comment]
Isn't it a single theme?

**Comment by sbrunner** [comment]
In the flush mode yes but not in the non flush mode...

**Comment by tsauerwein** [comment]
I am not sure I understand what you mean. Please adapt the branch as you want (you should have access rights).

**Comment by sbrunner** [comment]
> you should have access rights

I don't or I don't know how to do it ...

**Comment by tsauerwein** [comment]
See:
https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/
https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by sbrunner** [comment]
I do this but in ssh, it it's working only on https its not usable :-(

**Comment by tsauerwein** [comment]
You only have to click on "Use SSH":
ngeo-ssh

**Pull request closed without merging**